docs/reference/ingest/search-inference-processing.asciidoc

```diff
@@ -102,10 +102,13 @@ Here, you'll be able to:
 1. Choose a name for your pipeline.
   - This name will need to be unique across the whole deployment.
   If you want this pipeline to be index-specific, we recommend including the name of your index in the pipeline name.
+  - If you do not set the pipeline name, a default unique name will be provided upon selecting an ML trained model.
 2. Select the ML trained model you want to use.
+  - The model must be deployed before you can select it.
+  To begin deployment of a model, click the *Deploy* button.
```

Comment:
Some curated models (E5 and ELSER at this time) are pinned to the top of the list even if they are not deployed to the cluster. The user can select either of these, however the pipeline creation cannot continue unless the model is deploying/deployed/started (and so the Continue button is disabled).

The deployment can be started from this screen using the Deploy button, which begins the deployment of the selected model and enables the Continue button right away.


<img width="1438" alt="Screenshot 2023-12-28 at 11 54 42" src="https://github.com/elastic/elasticsearch/assets/14224983/f06161de-1197-4578-b956-a3e03fb5b367">

(This doesn't apply to a 3rd party model, which is already in the deployed state after the user uploads it to the cluster.)